cms/admin/settingsadmin.py

```diff
@@ -33,13 +35,13 @@ def wrapper(*args, **kwargs):
         info = self.model._meta.app_label, self.model._meta.model_name
 
         return [
-            re_path(r'^session_store/$',
+            path('session_store/',
                 self.session_store,
                 name='%s_%s_session_store' % info),
-            re_path(r'^cms-toolbar/$',
+            path('cms-toolbar/',
                 wrap(self.get_toolbar),
                 name='%s_%s_get_toolbar' % info),
-            re_path(r'^$',
+            path('',
                 wrap(self.change_view),
                 name='%s_%s_change' % info),
             re_path(r'^(.+)/$',
```

Comment:
Can this also be replaced by `re_path`?

Reply:
@fsbraun , Why not use `path` instead of `re_path`? Django official document prefers the use of  `path` also.
Can you tell me the reason why we need to update above `path` with `re_path`?

Reply:
Sorry, I meant: replace the last `re_path` in line 47 by `path`, too?

Reply:
Updated. Can you review again?

Reply:
Sure. It's already approved. Just thought you might have missed the last instance of `re_path`.  I'm also fine with keeping the `re_path`.  However, if you replace it, you'd also need to replace the regex... Probably something like `path("<path:id>/")`.

Reply:
@vipulnarang95  please remove the trailing `$` ,that will resolve the failing checks.

Reply:
@fsbraun  please review